python/PIK_maya_loader/scripts/PIKmenu/loader.py

```diff
@@ -0,0 +1,108 @@
+import  os
+import  sys
+import  json
+from    maya    import cmds
+from    maya    import mel
+from    maya    import OpenMaya as om
```

Comment:
Please don't create aliases in library names, full name is better than small names.

Aliases are usefull when 2 modules have similar functions and you can't load them at the same time, here it's useless.